lib/packages.cf

```diff
@@ -32,13 +32,21 @@ bundle common package_module_knowledge
     windows::
       "platform_default" string => "msiexec";
 @endif
+@if minimum_version(3.16.1)
+    termux::
+      "platform_default" string => "apt_get";
+@endif
 }
 
 body package_module apt_get
 {
     query_installed_ifelapsed => "$(def.package_module_query_installed_ifelapsed)";
     query_updates_ifelapsed => "$(def.package_module_query_updates_ifelapsed)";
     #default_options =>  {};
+@if minimum_version(3.16.1)
```

Comment:
@nickanderson so maybe ONLY have a guard here for 3.12.2 where interpreter was introduced and remove all others since termux hard-class will almost certainly never be defined?